src/Control/Monad/Bayes/Sampler/Lazy.hs

```diff
@@ -67,5 +71,44 @@ sampler m = newStdGen *> (runSampler m . randomTree <$> getStdGen)
 independent :: Monad m => m a -> m [a]
 independent = sequence . repeat
 
+-- | 'weightedsamples' runs a probability measure and gets out a stream of (result,weight) pairs
 weightedsamples :: Weighted Sampler a -> IO [(a, Log Double)]
 weightedsamples = sampler . independent . weighted
+
+wiener :: Sampler (Double -> Double)
```

Comment:
@reubenharry Do you have some example of usage of the Wiener process generation?
It is a hidden gem now.

Reply:
There are examples in LazyPPL (source code on bitbucket) but perhaps it should be removed in Monad-Bayes.

Reply:
resolved in #215